src/main/java/pokecube/core/blocks/maxspot/MaxTile.java

```diff
@@ -44,8 +45,8 @@ public boolean addForbiddenSpawningCoord()
     {
         if (this.getWorld() == null || this.getWorld().isRemote || !this.enabled) return false;
         final BlockPos pos = this.getPos();
-        return SpawnHandler.addForbiddenSpawningCoord(pos.getX(), pos.getY(), pos.getZ(), this.world.getDimension()
-                .getType().getId(), this.range, MaxTile.MAXSPOT);
+        return SpawnHandler.addForbiddenSpawningCoord(pos.getX(), pos.getY(), pos.getZ(), ((INBT) this.world.getDimensionType())
```

Comment:
This section needs some different adjustments, the addForbiddenSpawningCoord part needs changing to work with new dimension code